doc/Programming-guide/Your-first-Kaa-application/index.md

```diff
@@ -165,15 +165,15 @@ The following code block illustrates a simple desktop application that sends vir
 
 <div id="prep-c" class="tab-pane fade in active" markdown="1" >
 
-Before you start with C application code, some preparation is required.
+Before you start with C application code, some preparation is required:
 
-1. Install dependencies: CMake. On Ubuntu, you can install it using following commands.
+1. Install dependencies: CMake. On Ubuntu, you can install it using following command:
 
         sudo apt-get install cmake
 1. Create `kaa` directory and unpack C SDK
```

Comment:
I don't mind to fix it, but we have another task for this.
http://jira.kaaproject.org/browse/KAA-1282?jql=text%20~%20%22Review%20all%20C%20code%20samples%20in%20new%20Documentation%22